aplib.net-demo/Assets/Scripts/Configuration.cs

```diff
@@ -0,0 +1,277 @@
+using LevelGeneration;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Configuration : MonoBehaviour
+{
+    [SerializeField] private const float _maxPercentage = 0.7f;
+    [SerializeField] private const float _minPercentage = 0.2f;
+
+    [SerializeField] private SpawnableEnemies _spawnableEnemies;
+    [SerializeField] private SpawnableItems _spawnableItems;
+    [SerializeField] private GridConfig _gridConfig;
+
+    // Grid component variables.
+    public int MinGridSize { get { return 3; } }
+    public int MaxGridSize { get { return 100; } }
+
+    public int GridVolume { get; private set; }
+    public int GridSizeX { get; private set; }
+    public int GridSizeZ { get; private set; }
+
+    // Room component.
+    public int RoomAmount { get; private set; }
+
+    // Item ammo.
+    public int ItemAmmoAmount { get; private set; }
+
+    // Item health.
+    public int ItemHealthAmount { get; private set; }
+
+    // Item rage.
+    public int ItemRageAmount { get; private set; }
+
+    // Enemy melee.
+    public int EnemyMeleeAmount { get; private set; }
+
+    // Enemy ranged.
+    public int EnemyRangedAmount { get; private set; }
+
+    // Total items.
+    public int TotalItemCount { get; private set; }
+    // Total enemies.
+    public int TotalEnemyCount { get; private set; }
+
+    // Enemy component variables.
+    public bool TestingSwitch { get { return _test; } set { _test = value; } }
+    private bool _test = false;
```

Comment:
The default value of a boolean is false. Thus, `public bool TestingSwtich { get; set; }` would be equal.